modules/tinymce/src/plugins/fullpage/main/ts/core/FilterContent.ts

```diff
@@ -14,8 +14,8 @@ import * as Protect from './Protect';
 
 const each = Tools.each;
 
-const low = function (s) {
-  return s.replace(/<\/?[A-Z]+/g, function (a) {
+const low = function (s: string) {
```

Comment:
why not arrow functions?

Reply:
Just didn't happen to change them. Fixed that.